package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "expression-atlas-autocomplete",
-  "version": "2.0.0",
+  "version": "3.0.0-beta",
```

Comment:
Oooh. What do you class as beta then? At what point do you move out of beta?

Reply:
As soon as you approve it! 😄 I did my fair share of beta testing, but in principle that’s not enough.
What we usually do is to wait for a review to merge a feature/bugfix and then bump the version. We could’ve also done that and I could test this package with `npm pack`, but it’s usually a bit more cumbersome and I see no harm in releasing alpha/beta versions.